### ISSUE TITLE
Is there a way to hold the mousedown after a drag

### ISSUE THREAD

**Issue by alanwsmith** [asweigart/pyautogui#742]
I'm doing some screen recording of Scratch to make tutorials. The goal is to use pyautogui and screen recording to make short recording for every action. 

One nice to have feature would be for the mouse to want a bit before dropping each piece. Is there are "hold after drag" feature?

**Comment by SamiTesla** [comment]
You could tell it to wait by using delay
